hack/verify-license.py

```diff
@@ -0,0 +1,945 @@
+#!/usr/bin/env python
```

Comment:
Given that this is a golang community, writing it in python reduces its accessibility. What advantages from writing this in python justify us incurring this cost?

Reply:
Verify scripts are currently written in bash or python. Python has a built-in diffing library that makes this shorter. It could be rewritten in Go to make it standalone.